<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>249</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Список студентов</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить ученика</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbMath">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценка по математике:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMath">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbPhysics">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценка по физике:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePhysics">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbRussian">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценка по Русскому языку:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRussian">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwScore">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить ученика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;*перед изменением &lt;/p&gt;&lt;p&gt;надо выбрать студента&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveAll">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="acOpen"/>
    <addaction name="acSave"/>
    <addaction name="asSaveHow"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="acOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="acSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="asSaveHow">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
